README.md

```diff
@@ -7,13 +7,14 @@ PyMC-BART extends [PyMC](https://github.com/pymc-devs/pymc) probabilistic progra
 
 ## Installation
 
-PyMC-BART requires a working Python interpreter (3.8+). We recommend installing Python and key numerical libraries using the [Anaconda Distribution](https://www.anaconda.com/products/individual#Downloads), which has one-click installers available on all major platforms.
+PyMC-BART is available on Conda-Forge. To set up a suitable Conda environment, run
```

Comment:
Instructions for "a pure pip" option should still be available

Reply:
Done. Any further suggestions?